modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_renderer/init.jsp

```diff
@@ -30,6 +30,7 @@ java.lang.String namespace = GetterUtil.getString((java.lang.String)request.getA
 boolean persistDefaultValues = GetterUtil.getBoolean(String.valueOf(request.getAttribute("liferay-data-engine:data-layout-renderer:persistDefaultValues")));
 boolean persisted = GetterUtil.getBoolean(String.valueOf(request.getAttribute("liferay-data-engine:data-layout-renderer:persisted")));
 boolean readOnly = GetterUtil.getBoolean(String.valueOf(request.getAttribute("liferay-data-engine:data-layout-renderer:readOnly")));
+boolean submittable = GetterUtil.getBoolean(String.valueOf(request.getAttribute("liferay-data-engine:data-layout-renderer:submittable")));
```

Comment:
Please, sort this line. Meaning, put the dynamicAttributes in alphabetic order.

Reply:
Hey, @boton. It appears that you didn't sort this in your new push. Could you please make sure this is sorted too?

Reply:
It´s a mystery the `data-engine/data-engine-taglib/liferay-data-engine.xml` is sorted.
If you want I can cherry-pick this [commit](https://github.com/boton/liferay-portal/pull/223/commits/6d7a697d84335518a971cf81b0eb1e49a406fd80) but keep in mind that is automatically reverted running [`buildTaglibs`](https://github.com/boton/liferay-portal/pull/223/commits/90e8bb24a06b9c84dba61ef76d2d6202bd3985d4) .

Reply:
I see. Thanks for pointing that out, @boton. You can move this `dynamicAttributes` to the init-ext, since it makes more sense to leave in the init just the generated ones. Does that sound good?

Reply:
Oh. I get it now. It's being added by the task regardless. It makes sense then to leave it as is.
We can resolve this conversation.

Thanks for your patience, @boton.

Reply:
Thanks to you @rodrigopaulino